onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -5,8 +5,10 @@
 # --------------------------------------------------------------------------
 
 from ....proto import onnx_proto
-from ...common._apply_operation import apply_exp, apply_reshape, apply_sub
+from ...common._apply_operation import apply_exp, apply_reshape, apply_sub, apply_max, apply_min
```

Comment:
Did max and min op need broadcasting support?

Reply:
It would have been better.